apps/src/code-studio/pd/workshop_dashboard/local_summer_workshop_survey_results.jsx

```diff
@@ -102,6 +116,43 @@ const LocalSummerWorkshopSurveyResults = React.createClass({
     );
   },
 
+  renderRow(row, i) {
+    let scoreCells;
+    let thisWorkshopData = this.state.thisWorkshop[row['key']];
+
+    if (this.state.facilitatorBreakdown && typeof thisWorkshopData === 'object') {
+      scoreCells = this.state.facilitatorNames.map((facilitator, i) => {
+        return (
+          <td key={i}>
+            {this.renderScore(row, thisWorkshopData[facilitator])}
+          </td>
+        );
+      });
+
+      scoreCells.unshift((<td key={this.state.facilitatorNames.length}/>));
+    } else {
+      scoreCells = [(
+        <td key={0}>
+          {this.renderScore(row, thisWorkshopData)}
+        </td>
+      )];
+
+      if (this.state.facilitatorBreakdown) {
+        _.times(this.state.facilitatorNames.length, (i) => {
+          scoreCells.push((<td key={i + 1}/>));
+        });
```

Comment:
Yup - they are empty cells. These are for the organizer view - we leave the cells under facilitators blank for non-facilitator breakdown questions, and leave the cells under this workshop blank for facilitator breakdown questions